CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+
+## Getting Started
+
+General information about setting up your Python environment, testing modules,
+Ansible coding styles, and more can be found in the [Ansible Community Guide](
+https://docs.ansible.com/ansible/latest/community/index.html).
+
+Information about boto library usage, module utils, testing, and more can be
+found in the [AWS Guidelines](https://docs.ansible.com/ansible/devel/dev_guide/platforms/aws_guidelines.html)
+documentation.
+
+## AWS Collections
+
+There are two related collections containing AWS content (modules and plugins).
+
+### amazon.aws
+This collection contains the `module_utils` (shared libraries) used by both collections.
+Content in this collection is included downstream in Red Hat Ansible Automation Platform.
+
+Code standards, test coverage, and other supportability criteria may be higher in this collection. 
+
+### community.aws
+This collection contains modules and plugins contributed and maintained by the Ansible AWS
+community.  The `community.aws` collection is tested and generally assured to work in
+conjunction with `amazon.aws`.
+
+New modules and plugins developed by the community should be proposed to `community.aws`.
+Content in this collection that is stable and meets other acceptance criteria has the potential
+to be promoted and migrated into `amazon.aws`.
+
+## Submitting Issues
+All software has bugs, and the `amazon.aws` collection is no exception. When you find a bug, 
+you can help tremendously by [telling us about it](https://github.com/ansible-collections/amazon.aws/issues/new/choose).
+
+If you should discover that the bug you’re trying to file already exists in an issue, 
+you can help by verifying the behavior of the reported bug with a comment in that 
+issue, or by reporting any additional information
+
+## Pull Requests
```

Comment:
IIRC the main Ansible docs included a section on how to use the integration test suites, would it be worth linking from here over to that documentation?

Building out a fresh test suite can be a little daunting, would it be worth suggesting people come visit us on IRC if they don't know how to start out?  I kinda wonder if we should have a specific example test suite people could start with.